parsely/src/test/java/com/parsely/parselyandroid/FlushQueueTest.kt

```diff
@@ -0,0 +1,247 @@
+package com.parsely.parselyandroid
+
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.test.runCurrent
+import kotlinx.coroutines.test.runTest
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@OptIn(ExperimentalCoroutinesApi::class)
+@RunWith(RobolectricTestRunner::class)
+class FlushQueueTest {
+
+    private lateinit var sut: FlushQueue
```

Comment:
Similar to my previous comment, do we need to use a property or could we use a local variable instead?